chef/cookbooks/network/recipes/default.rb

```diff
@@ -249,6 +239,63 @@ def kill_nic(nic)
     our_iface = br
     net_ifs << our_iface.name
   end
+  if network["add_ovs_bridge"]
+    bridge = node[:network][:networks][name][:bridge_name] || "br-#{name}"
+
+    node[:network][:ovs_pkgs].each do |pkg|
+      p = package pkg do
+        action :nothing
+      end
+      p.run_action :install
+    end
+
+    unless ::File.exists?("/sys/module/#{node[:network][:ovs_module]}")
+      ::Kernel.system("modprobe #{node[:network][:ovs_module]}")
+    end
+
+    s = service node[:network][:ovs_service] do
+      service_name node[:network][:ovs_service]
+      supports status: true, restart: true
+      action [:nothing]
+    end
+    s.run_action :enable
+    s.run_action :start
```

Comment:
Yeah, right. And I actually had code already to only call it once. Most have lost it during a rebase or something. :(